src/ui/ons/ONSSelect.svelte

```diff
@@ -1,15 +1,41 @@
 <script>
-  export let label;
-  export let id;
-  export let name;
+  import ONSError from "./partials/ONSError.svelte";
+  export let label, id, name, renderError, selectValue, onClick;
+  export let errorText = "Select [whatever topic the drop-down is asking for]";
   export let options = [];
+
+  function disablesSelectedOption(options) {
+    let selectOption = document.getElementById(id);
+    if (selectOption.options[selectOption.options.selectedIndex].selected) {
+      options.find((item) => {
+        if (item.value === selectOption.options[selectOption.options.selectedIndex].value) {
+          item.disabled = true;
+        } else {
+          item.disabled = false;
+        }
+      });
+    }
+  }
```

Comment:
This is unnecessary and may cause some accessibility or parsing issues. We would only disable a value for something like "Please select".

Reply:
So then, should I remove the function and use a data structure like this : `let cities=[
    { value: "placeholder", label: "Please select",disabled:true},
    { value: "london", label: "London",},
    { value: "bristol", label: "Bristol",},
    { value: "oxford", label: "Oxford",},
  ]` ?

Reply:
Yep, that seems like what we want there.

Reply:
Done.